examples/build-reduced-model.py

```diff
@@ -0,0 +1,39 @@
+import pinocchio as pin
+import example_robot_data
```

Comment:
It would be nice to not rely on this package for the examples. In the urdf example for Python, you can copy/past the loading of the robot to load the UR5.